app/components/HoverPreview/HoverPreview.tsx

```diff
@@ -143,7 +141,7 @@ function HoverPreviewInternal({ element, id, onClose }: Props) {
                     />
                   ) : data.type === UnfurlType.Document ? (
                     <HoverPreviewDocument
-                      id={id}
+                      id={data.meta.id}
```

Comment:
Summary didn't change while hovering among adjacent elements. This change fixed the problem, but I hope it's the correct way to fix this problem :crossed_fingers: 

https://github.com/outline/outline/assets/15694566/f95f8130-b794-4322-be9d-cb81f98a4c59

cc: @tommoor